src/ashley/api/views.py

```diff
@@ -90,3 +92,12 @@ def update(self, request, pk=None):
             raise PermissionDenied()
 
         return Response({"success": True}, status=status.HTTP_200_OK)
+
+
+# pylint: disable=too-many-ancestors
+class ImageUploadApiView(viewsets.ModelViewSet):
+    """Viewset for the API of the Image object."""
+
+    queryset = UploadImage.objects.all()
+    serializer_class = UploadImageSerializer
+    permission_classes = [IsAuthenticated]
```

Comment:
When a `UploadImage` object is created or updated, the  `poster` field should be retrieved from the authenticated user that is doing the request. Or at least you should ensure that it is the same.

We don't want to trust the user input on this one.